src/cmap/handshake/client_metadata.ts

```diff
@@ -0,0 +1,94 @@
+import * as os from 'os';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const NODE_DRIVER_VERSION = require('../../../package.json').version;
+
+/**
+ * FaaS environment metadata.
+ * @public
+ */
+export interface FaasMetadata {
```

Comment:
```suggestion
export interface FAASMetadata {
```

we went with all caps for OIDC, just think acronyms can stay in caps.

Reply:
```suggestion
export interface FaaSMetadata {
```

or..